missing a semicolon but this is in the `if big endian` block
diff --git a/lib/dnsdb/include/dnsdb/zdb-zone-path-provider.h b/lib/dnsdb/include/dnsdb/zdb-zone-path-provider.h
index 3f8808b..eb8ecad 100644
--- a/lib/dnsdb/include/dnsdb/zdb-zone-path-provider.h
+++ b/lib/dnsdb/include/dnsdb/zdb-zone-path-provider.h
@@ -112,7 +112,7 @@ union zdb_zone_info_provider_data
     struct { u16 _u16_0, _u16_1, _u16_2, _u16; };
     struct { u32 _u32_0, _u32; };
     u64 _u64;
-    struct { ya_result _result_0, _result; }
+    struct { ya_result _result_0, _result; };
     void *_ptr;
     zdb_zone_path_provider_buffer _buffer;
 };
